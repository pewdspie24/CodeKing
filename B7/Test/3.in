10
5 7263
574 191
591 769
294 334
832 341
178 683
9 831
792 818
860 540
584 137
743 371
612 214
216 422
575 359
588 495
108 789
6 5080
541 688
730 905
709 281
105 778
684 911
564 889
1 3780
528 869
2 1245
425 167
359 492
4 6332
551 365
912 127
434 588
616 957
10 5305
377 960
109 739
197 124
684 368
783 496
234 448
366 902
812 904
415 257
666 959
7 7406
946 873
118 376
600 573
758 337
543 797
460 212
332 547
6 171
394 128
254 878
212 143
353 218
557 704
241 426
8 7211
259 757
787 239
398 992
440 187
861 873
200 495
850 694
663 188
