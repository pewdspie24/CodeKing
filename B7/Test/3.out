15
1
12
MISSION FAILED!
5
8
16
17
1
13
